### PR TITLE
Search

### PR DESCRIPTION
**Pull request by Jareechang**
Hey @cynthialee @lkysow  @amoghsa  @amyngai. I added a basic 'filter' functionality. The numbers on the map do change when I click a few organizations. I will need to check if it is actually filtering things right. 

I assume the results panel on the right will change on selection of filters?

### CODE REVIEW

**Comment by amyngai** [comment]
correct!

**Comment by Jareechang** [comment]
cool! i'll try to look into that!

File: src/angular/Views/filters.html [hunk 1]
```diff
@@ -15,48 +15,26 @@
...
-            </div>
-          </label>
+
+        <!-- use this as a skeleton for a ng-repeat once we figure out getting the organization list -->
+        <div ng-repeat="organization in organizations">
+          <div class="form-group">
+            <label for="inputReferral" class="col-xs-12 col-md-6 control-label">
+              {{organization}}
+              <div class="pull-right">
+                <img src="..." />
```
Comment (cynthialee) [inline src/angular/Views/filters.html:31]: Maybe we should comment this out until we put this in (#75). Otherwise, it'll be a broken image until then.

**Comment by cynthialee** [comment]
Looks good to me :) I'll pull - anything that needs to change we can do on top.

**Pull request merged**